### PR TITLE
add keyboard shortcut for adding a comment

### LINKED ISSUE #5741: It should be possible to add a comment from anywhere
As noted in https://github.com/RecordReplay/devtools/issues/5527 , we need a keyboard shortcut that will add a shortcut in all scenarios.

* If the user is paused at a point in time, we use time
* If the user is paused at an execution point, we use that

Once we have the back-end logic wired up to a keyboard shortcut, the remaining work is just designing a button and slapping it into the Comments pane.

(For now, the shortcut can be shift-C, but this can be modified as we go)

### PR DESCRIPTION
**Pull request by souporserious**
Adds a "Shift+C" global keyboard shortcut for adding a comment from anywhere in the application.

closes #5741

### CODE REVIEW

**Review by jasonLaster — APPROVED** [review]

File: src/ui/actions/comments.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ export function updatePendingCommentContent(content: string): UpdatePendingComme
 export function createComment(
   time: number,
   point: string,
-  user: User,
+  user: User, // TODO: user is no longer required and should be removed
```
Comment (souporserious) [inline src/ui/actions/comments.ts:50]: Chatted with Josh, `user` is no longer required so we should safely be able to refactor this and get rid of it in the future.

File: src/ui/components/KeyboardShortcuts.tsx [hunk 2]
```diff
@@ -113,6 +120,29 @@ function KeyboardShortcuts({
       toggleQuickOpen(query, project);
     };
 
+    const addComment = (e: KeyboardEvent) => {
+      if (!e.target || !isEditableElement(e.target)) {
+        e.preventDefault();
+        const commentContent: any = pendingComment?.comment.content;
+        const hasComment =
```
Comment (souporserious) [inline src/ui/components/KeyboardShortcuts.tsx:127]: This feels gross, but not sure how else to determine when Tiptap has content 😕

File: src/ui/components/KeyboardShortcuts.tsx [hunk 3]
```diff
@@ -113,6 +120,29 @@ function KeyboardShortcuts({
...
+          commentContent === ""
+            ? false
+            : Boolean(commentContent?.content && commentContent.content[0].content);
```
Comment (souporserious) [inline src/ui/components/KeyboardShortcuts.tsx:130]: This feels gross, but not sure how else to determine when Tiptap has content 😕

**Comment by souporserious** [comment]
@jonbell-lot23 when you get a chance can you give this a quick test and let me know if it's working how you expected? Also, not sure who to ask, but should we track this event the same as the other `addComment` button using `trackEvent("breakpoint.add_comment")`?

**Review by jonbell-lot23 — APPROVED** [review]
Looks good!

Let's track a follow-on ticket for these items:

* Let's add a comment button into the header of the comments pane
* Let's add "Add comment" to the command palette. Maybe we don't know the shortcut just yet, we're still deciding if it should be shift-C

Nice work, woo!

**Pull request merged**